test/src/main/java/org/axonframework/test/matchers/NonTransientFieldsFilter.java

```diff
@@ -31,8 +27,8 @@ public class NonTransientFieldsFilter implements FieldFilter {
     private static final NonTransientFieldsFilter INSTANCE = new NonTransientFieldsFilter();
 
     @Override
-    public boolean accept(Field field) {
-        return !ReflectionUtils.isTransient(field);
+    public boolean accept(String field) {
+        return false;
```

Comment:
The actual transient fields filtering happens in the calling class. This is why this return here. It's confusing, though. Is there a better solution?